src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -136,6 +118,22 @@ class HoldRequest extends React.Component {
   }
 
   /**
+   * modelDeliveryLocationName(prefLabel, shortName)
+   * Renders the names of the radio input fields of delivery locations except EDD.
+   *
+   * @param {String} prefLabel
+   * @param {String} shortName
+   * @return {String}
+   */
+  modelDeliveryLocationName(prefLabel, shortName) {
+    if (prefLabel && typeof prefLabel === 'string' && shortName) {
+      return `${shortName} - ${prefLabel.split(' - ')[1]}`;
```

Comment:
I accidentally made changes on `dev-eb-deploy` so I branched it out to a hot fix branch and then merged that hot fix branch back to my feature branch. Turns out it is not a perfect strategy to fix it ugh